baton/modules/twilio/twilio.go

```diff
@@ -0,0 +1,179 @@
+package twilio
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/gorilla/mux"
+)
+
+type Twilio struct {
+	UserId, ApiKey string
+}
+
+var (
+	smsCallbacks  = make([]callback, 0)
+	callCallbacks = make([]callback, 0)
+)
```

Comment:
We've discussed this before, but I think it's important that the application itself is stateless so we can have multiple instances running (even if just for redundancy).

We can focus on that later though.